src/content/endpoints.md

```diff
@@ -1 +1,3 @@
 Below is documentation in Swagger. You can also download our [Postman Collections](resources/NYC_Benefits_Eligibility_API_Postman_Collection.zip) for testing.
+
+To generate a valid JSON payload for the `/eligibilityPrograms` endpoint, you can use the [Request Builder](request-builder)
```

Comment:
@kimpenguin @abourne1 Since this sentence is under the /eligibilityPrograms endpoint in the swagger docs, we can probably move this particular instance of the sentence to "Making a Request" as the last sentence of the paragraph "Below is the schema for each type. For a detailed description of how to structure the data for each endpoint, please refer to the endpoint documentation."

Reply:
So it could become this;

"Below is the schema for each type. For a detailed description of how to structure the data for each endpoint, please refer to the endpoint documentation. To generate a valid JSON payload (for the `/eligibilityPrograms` endpoint), you can use the [Request Builder](request-builder)."